### PR TITLE
Task 52619 send mail active account reset password

### PR DESCRIPTION
**Pull request by luctq-0138**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52014)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by luctq-0138** [comment]
ready

File: app/controllers/account_activations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class AccountActivationsController < ApplicationController
+  before_action :find_user, only: %i(edit)
+  def edit
```
Comment (ngocvt-0484) [inline app/controllers/account_activations_controller.rb:4]: thêm phía trên dòng trống

File: app/controllers/account_activations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class AccountActivationsController < ApplicationController
+  before_action :find_user, only: %i(edit)
```
Comment (ngocvt-0484) [inline app/controllers/account_activations_controller.rb]:
```suggestion
  before_action :find_user, only: :edit
```

**Comment by luctq-0138** [comment]
ready

**Pull request merged**